packages/apps/docs/src/components/BottomPageSection/components/EditPage.tsx

```diff
@@ -1,18 +1,29 @@
 import { Button } from '@kadena/react-ui';
 
+import { analyticsEvent, EVENT_NAMES } from '@/utils/analytics';
+import { useRouter } from 'next/router';
 import React, { FC } from 'react';
 
 interface IProps {
   editLink?: string;
 }
 
 export const EditPage: FC<IProps> = ({ editLink }) => {
+  const router = useRouter();
+
   if (!editLink) return null;
+  const onClick = async (
+    event: React.MouseEvent<HTMLElement>,
+  ): Promise<void> => {
+    event.preventDefault();
+    event.stopPropagation();
+    analyticsEvent(EVENT_NAMES['click:edit_page']);
+    await router.push(editLink);
+  };
```

Comment:
sorry! my bad! I thought it's an internal route! But it seems Github link. So, we don't need `useRouter` and it also needs to open in a new tab that's also missing with the code change